src/main/java/uk/nhs/hee/tis/trainee/sync/service/TcsSyncService.java

```diff
@@ -53,18 +54,22 @@
   private static final String TABLE_PLACEMENT = "Placement";
   private static final String TABLE_PROGRAMME_MEMBERSHIP = "ProgrammeMembership";
   private static final String TABLE_CURRICULUM = "Curriculum";
-
-  private static final Map<String, String> TABLE_NAME_TO_API_PATH = Map.of(
-      TABLE_CONTACT_DETAILS, "contact-details",
-      TABLE_GDC_DETAILS, "gdc-details",
-      TABLE_GMC_DETAILS, "gmc-details",
-      TABLE_PERSON, "basic-details",
-      TABLE_PERSON_OWNER, "person-owner",
-      TABLE_PERSONAL_DETAILS, "personal-info",
-      TABLE_QUALIFICATION, "qualification",
-      TABLE_PLACEMENT, "placement",
-      TABLE_PROGRAMME_MEMBERSHIP, "programme-membership",
-      TABLE_CURRICULUM, "curriculum"
+  private static final String TABLE_PLACEMENT_SPECIALTY = "PlacementSpecialty";
+  private static final String TABLE_SPECIALTY = "Specialty";
+
+  private static final Map<String, String> TABLE_NAME_TO_API_PATH = Map.ofEntries(
+      Map.entry(TABLE_CONTACT_DETAILS, "contact-details"),
+      Map.entry(TABLE_GDC_DETAILS, "gdc-details"),
+      Map.entry(TABLE_GMC_DETAILS, "gmc-details"),
+      Map.entry(TABLE_PERSON, "basic-details"),
+      Map.entry(TABLE_PERSON_OWNER, "person-owner"),
+      Map.entry(TABLE_PERSONAL_DETAILS, "personal-info"),
+      Map.entry(TABLE_QUALIFICATION, "qualification"),
+      Map.entry(TABLE_PLACEMENT, "placement"),
+      Map.entry(TABLE_PROGRAMME_MEMBERSHIP, "programme-membership"),
+      Map.entry(TABLE_CURRICULUM, "curriculum"),
+      Map.entry(TABLE_PLACEMENT_SPECIALTY, "placement-specialty"),
+      Map.entry(TABLE_SPECIALTY, "specialty")
```

Comment:
Are these ever used?  I can't see any calls to the this class from the enricher for a `PlacementSpecialty` or `Specialty`.

Reply:
I deleted these, I don't think they're used.

Reply:
Sorry, I think that was speculative code I added before a fire-fire that I never got back to